modules/primer-buttons/README.md

```diff
@@ -245,6 +245,21 @@ Use `.hidden-text-expander` to indicate and toggle hidden text.
 
 You can also make the expander appear inline by adding `.inline`.
 
+
+#### Details element summary buttons
```

Comment:
Suggest changing to `Using button styles with the details summary element` so that it's more "button focussed" 😏